packages/safety-suite-api/src/api/permission/grants.ts

```diff
@@ -157,3 +158,82 @@ export const deleteGrant = (
       query
     }
   });
+
+/**
+ * Start async users export jobs.
+ *
+ * @param customerAlias - customer alias of desired users.
+ * @param apiOptions - Optional options for runApi.
+ * @returns - Users export job.
+ */
+export const runUsersExportJob = (
+  customerAlias: string,
+  apiOptions?: ApiOptions
+): Request<ApiResponse<QueryExportJob>> =>
+  runApi({
+    method: 'POST',
+    urlRoot: 'permissionUrlRoot',
+    route: `/api/grants/export?customerAlias=${customerAlias}`,
```

Comment:
Pass the customer alias in a query object instead. Example here: [src/api/claims/claims.ts](https://github.com/idelic-inc/idelic-js/blob/master/packages/safety-suite-api/src/api/claims/claims.ts)